core/api/src/domain/accounts/index.types.d.ts

```diff
@@ -66,17 +66,17 @@ interface IAccountTxVolumeLimitChecker {
 interface IAccountTxVolumeRemaining {
   intraLedger: (args: {
     priceRatio: WalletPriceRatio
-    walletVolumes: TxBaseVolumeAmount<WalletCurrency>[]
+    outWalletVolumes: PaymentAmount<WalletCurrency>[]
```

Comment:
in theory PaymentAmount should be only used for payments...not sure about this type change... cc @bodymindarts

Reply:
This is really `WalletAmount`. It's a change we discussed a long time ago but never actually refactored in. We decided there was no need for a distinction among things like `PaymentAmount`, `BalanceAmount` etc.